CHANGELOG.rst

```diff
@@ -1,8 +1,10 @@
 Changelog
 =========
 
-0.0.1.dev3 (2018-05-22)
------------------------
+0.0.1.dev3 (2018-05-22 / 2018-05-25)
```

Comment:
Two things: 1. I think this should be edited once a release is made (aka not in this PR/branch), and 2. Once we do make a release, I think we shouldn't merge the two histories like this. Rather, just have a line item under the released version saying that the repos have been merged. We can point to the change log of the janitor GCP if folks are curious.

Reply:
Also FWIW we can't re-release a version since PyPI does not allow that. We are going to have to bump the release.

Reply:
Yeah I know this became a mess.  I think you're right.  I'll also add a commit to bump the version.